lib/rex/proto/crypto_asn1.rb

```diff
@@ -46,4 +46,20 @@ class EnrollmentNameValuePair < RASN1::Model
       bmp_string(:value)
     ]
   end
+
+  # see: [[MS-WCCE]: 2.2.2.7.7.4 szOID_NTDS_CA_SECURITY_EXT](https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-wcce/e563cff8-1af6-4e6f-a655-7571ca482e71)
+  class NtdsCaSecurityExt < RASN1::Model
```

Comment:
It'd be great to add a unit test for this model or update the existing ms_icpr_spec with another `context` block for when alt_sid is provided

Just to help future proof things